In [5]:
require("haven")
datos <- read_spss("Brillo_Resist_Opac.sav")
datos

extru aditivo resist opaci brillo
1  1     1       6.5    4.4    9.5  
2  1     1       6.2    6.4    9.9  
3  1     1       5.8    3.0    9.6  
4  1     1       6.5    4.1    9.6  
5  1     1       6.5    0.8    9.2  
6  1     2       6.9    5.7    9.1  
7  1     2       7.2    2.0   10.0  
8  1     2       6.9    3.9    9.9  
9  1     2       6.1    1.9    9.5  
10 1     2       6.3    5.7    9.4  
11 2     1       6.7    2.8    9.1  
12 2     1       6.6    4.1    9.3  
13 2     1       7.2    3.8    8.3  
14 2     1       7.1    1.6    8.4  
15 2     1       6.8    3.4    8.5  
16 2     2       7.1    8.4    9.2  
17 2     2       7.0    5.2    8.8  
18 2     2       7.2    6.9    9.7  
19 2     2       7.5    2.7   10.1  
20 2     2       7.6    1.9    9.2

In [10]:
# apartado a)
res <- datos$resist
opa <- datos$opaci
bri <- datos$brillo
ext <- datos$extru
adi <- datos$aditivo
reg <- lm(opa~ext+adi)
summary(reg)
# no tienen influencia significativa ni individualmente ni 
#    conjuntamente


Call:
lm(formula = opa ~ ext + adi)

Residuals:
   Min     1Q Median     3Q    Max 
-2.675 -1.903  0.015  1.183  3.825 

Coefficients:
            Estimate Std. Error t value Pr(>|t|)
(Intercept)   2.0150     1.9620   1.027    0.319
ext           0.2900     0.9002   0.322    0.751
adi           0.9900     0.9002   1.100    0.287

Residual standard error: 2.013 on 17 degrees of freedom
Multiple R-squared:  0.07171,	Adjusted R-squared:  -0.0375 
F-statistic: 0.6566 on 2 and 17 DF,  p-value: 0.5313


In [28]:
# apartado b)
reg2 <- lm(res~ext+adi)
summary(reg2)
# si tienen influencia significativa conjuntamente (p.valor 0.0005515)


Call:
lm(formula = res ~ ext + adi)

Residuals:
    Min      1Q  Median      3Q     Max 
-0.5850 -0.2075  0.0650  0.2150  0.5150 

Coefficients:
            Estimate Std. Error t value Pr(>|t|)    
(Intercept)   5.3150     0.3140  16.926 4.49e-12 ***
ext           0.5900     0.1441   4.095 0.000755 ***
adi           0.3900     0.1441   2.707 0.014960 *  
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 0.3222 on 17 degrees of freedom
Multiple R-squared:  0.5863,	Adjusted R-squared:  0.5377 
F-statistic: 12.05 on 2 and 17 DF,  p-value: 0.0005515


In [26]:
# apartado b3)
reg3 <- lm(res~ext+adi+ext*adi)
summary(reg3)
anova(reg3,reg2)
# de acuerdo a este test (anidados), nos quedamos con reg2 
#  (observese que p-valor de la ANOVA es grande)


Call:
lm(formula = res ~ ext + adi + ext * adi)

Residuals:
   Min     1Q Median     3Q    Max 
-0.580 -0.205  0.060  0.220  0.520 

Coefficients:
            Estimate Std. Error t value Pr(>|t|)    
(Intercept)   5.3600     0.7425   7.219 2.05e-06 ***
ext           0.5600     0.4696   1.193    0.250    
adi           0.3600     0.4696   0.767    0.454    
ext:adi       0.0200     0.2970   0.067    0.947    
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 0.332 on 16 degrees of freedom
Multiple R-squared:  0.5864,	Adjusted R-squared:  0.5089 
F-statistic: 7.563 on 3 and 16 DF,  p-value: 0.00227


Res.Df RSS    Df Sum of Sq F           Pr(>F)   
1 16     1.7640 NA     NA             NA        NA
2 17     1.7645 -1 -5e-04    0.004535147 0.9471426

El modelo elegido es 

$$
\hat{y} = 5.3150 + 0.5900 z_{11} + 0.3900 z_{21}
$$
donde $y$ es la resistencia y las variables indicador son las siguientes:

* $z_{11} = 1$ si extru = 1 y zero en otro caso

* $z_{21} = 1$ si aditivo = 1 y zero en otro caso

Nota: apartados b1) y b2) ya respondidos con lo escrito ahora mismo

In [24]:
# apartado c)
reg4 <- lm(bri~ext+adi)
summary(reg4)
# si tienen influencia estadisticamente significativa conjuntamente 
#    ya que el p-valor del contraste F es 0.01812.


Call:
lm(formula = bri ~ ext + adi)

Residuals:
   Min     1Q Median     3Q    Max 
-0.645 -0.355  0.035  0.225  0.865 

Coefficients:
            Estimate Std. Error t value Pr(>|t|)    
(Intercept)   9.5550     0.4211  22.693 3.76e-14 ***
ext          -0.5100     0.1932  -2.640   0.0172 *  
adi           0.3500     0.1932   1.812   0.0877 .  
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 0.432 on 17 degrees of freedom
Multiple R-squared:  0.3762,	Adjusted R-squared:  0.3028 
F-statistic: 5.125 on 2 and 17 DF,  p-value: 0.01812
